compose.yaml

```diff
@@ -66,6 +66,7 @@ services:
             - README.md
 
   divviup_api:
+    image: ${DIVVIUP_API_IMAGE:-us-west2-docker.pkg.dev/janus-artifacts/divviup-api/divviup_api:0.3.9}
```

Comment:
This PR isn't quite ready to go, because the default image here is not in a public repository. I'm working on separate changes to enable `docker-release.yml` to push to `us-west2-docker.pkg.dev/divviup-artifacts-public` and will update this URL to point there before merging this.